### PR TITLE
Sqli, Sream Errors, Temp Xss function

### PR DESCRIPTION
**Pull request by motters**
I have fixed the sqli issues in 5 modules (aliases, apache_admin, backup_admin, backupmgr, client_notices)

I have also fixed a few scream issues in multiple places. the panel should now be 100% error free.

I have started the xss class though is **incomplete** I cant exclude it from the pull that's all.

### CODE REVIEW

**Comment by motters** [comment]
Oh ye and fixed a module admin issue where xml function was receiving empty values

**Comment by allebb** [comment]
Just make sure you close all the bugs in the bug tracker mate that these fixes relate too :) - So we know what has been fixed and what else still needs to be done :)

Also - I see you didn't 'pull' the latest code from the Github repo as your changes to the client_notice module controller doesn't contain my latest CSFR code so just make sure in future you 'pull' as often as possible so we don't have to re-add code back in afterwards :)

Right I'll now merge these changes :)

Cheers mate,
Bobby

**Comment by allebb** [comment]
Motters - Just trying to merge the changes now... Github is saying that it cannot merge the changes (as you have not pulled my latest changes recently) - Would be a good idea if you can do a 'pull request' from the master (to update your repositoy) and hopefully Netbeans will merge my changes locally on your machine and then if you could open a new pull request that would be great as otherwise I'm going to have to go through all the code and manually merge it on my local machine which would be a lot quicker if you just 'pull' the latest from my mater and then open a new pull request 

Remember I've now given you access to the zpanelx repo so you can commit directly if you want... up to you :)

Cheers mate,
Bobby

**Comment by motters** [comment]
Are ok pal will do that merge thing now :)

Yeye will start doing that pal just i started these a bit ago in my repo is all :)

**Comment by allebb** [comment]
Ok mate cool :)

**Comment by motters** [comment]
There a shit load of merge issues. I'll sort them out them I'll have to test the modules again.

I'll try and get it sorted tonight but might take till tomorrow. Bollocks!

**Comment by allebb** [comment]
OK mate... I know, I've had real horrible merges to do in the past lol.... Its sunday evening I'd relax mate and just do it during the week :)

Great work though so far :)

File: modules/moduleadmin/code/controller.ext.php [hunk 1]
```diff
@@ -247,37 +247,37 @@ static function getIsShowModuleInfo() {
 
     static function getModuleInfoName() {
         global $controller;
-        $info = ui_module::GetModuleXMLTags($controller->GetControllerRequest('URL', 'showinfo'));
```
Comment (allebb) [inline modules/moduleadmin/code/controller.ext.php]: BTW mate... Although this fix suppresses the errors appearing when full PHP debug mode is enabled this doesn't fix it as the ?showinfo parameter is parsed and used when the module name  is clicked a new screen then displays information about the module of which name is passed in the showinfo parameter so I'll deal with this and the fix just requires an isset() wrapper to check if the user has clicked... I'll fix this now and commit the fix for this module. What happens with your current fix for this is that whatever module name is clicked the 'Module information' screen always shows information of the Module Admin module and not the individual module :)

Comment (motters) [inline modules/moduleadmin/code/controller.ext.php]: Are ok mate sorry about that :/  Cheers for check over things :)

Comment (allebb) [inline modules/moduleadmin/code/controller.ext.php]: No worries mate :) - You'll soon get the hang of it :)

**Comment by motters** [comment]
Ok hopefully i have solved it! I'll give it a test now before there's any more merges and i have to do it again lol

**Comment by allebb** [comment]
Ok mate  so you want me to merge now or wait for you to let me know your happy with the changes etc?

**Comment by motters** [comment]
Give us like 5 mins pal just testing now

**Comment by allebb** [comment]
Yep,  no worries mate :) - I won't merge until you reply here :)

**Comment by motters** [comment]
Alright pal i've tested the effected modules that had merge issues and they all work fine.

Merge when ever you want pal :D

**Pull request merged**